cme/lms/templates/header.html

```diff
@@ -12,18 +12,21 @@
   <img src="${static.url('images/stanford-S-logo.png')}" height="35" alt="Home Page" />
 </%block>
 
-% if course:
-    % if UserProfile.has_registered(user):
-        <%block name="navigation_global_links_authenticated">
-            %if settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD','') and user.is_staff:
-                <li class="item">
-                    ## Translators: This is short for "System administration".
-                    <a href="${reverse('sysadmin')}">${_("Sysadmin")}</a>
-                </li>
-            %endif
-        </%block>
-    %endif
-%else:
-    <%block name="navigation_global_links"></%block>
-    <%block name="navigation_other_global_links"></%block>
+% if UserProfile.has_registered(user):
+    <%block name="navigation_global_links_authenticated">
+        %if settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD','') and user.is_staff:
+            <li class="item">
+                ## Translators: This is short for "System administration".
+                <a href="${reverse('sysadmin')}">${_("Sysadmin")}</a>
+            </li>
+        %endif
+    </%block>
+% else:
+    <%block name="navigation_global_links">
```

Comment:
Have you taken a look at `./lms/templates/navigation.html` in the platform [1]?

It looks like the register link is originally set in `navigation_other_global_links`, not `navigation_global_links`.

I think the default logic actually gives us what we want.
Assuming so, let's remove this override and drop the one below too.

[1] https://github.com/Stanford-Online/edx-platform/blob/master/lms/templates/navigation.html#L154-L171